assets/src/app/videos/videos.component.html

```diff
@@ -80,6 +80,24 @@ <h1>
           (click)="videoid = ''; updateVideos()" >clear</mat-icon>
     </mat-form-field>
   </div>
+  <div>
+    <mat-form-field>
+      <mat-select
+        class="page-size-select"
+        placeholder="Nombre de vidéos"
+        [(ngModel)]="pageSize"
+        (selectionChange)="updateVideos()"
+        >
+        <mat-option
+          *ngFor="let size of pageSizes"
+          [value]="size"
+          class="page-size-option"
+          >
+          {{ size }}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
```

Comment:
use pageSizeOptions of [MatPaginator](https://material.angular.io/components/paginator/api) to configure it directly on the element itself